src/bun.js/webcore/response.zig

```diff
@@ -1551,6 +1556,10 @@ pub const Fetch = struct {
             hostname: ?[]u8 = null,
             memory_reporter: *JSC.MemoryReportingAllocator,
             check_server_identity: JSC.Strong = .{},
+            ca: ?[]const u8 = null,
```

Comment:
We could use empty strings here instead of optional strings to indicate nullable. ?[]const u8 costs 32 bytes where []const u8 costs 16

Reply:
32 bytes?